docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.8'
+
+services:
+
+  yaptide_flask:
+    build: .
+    image: yaptide_flask
+    container_name: yaptide_flask
+    ports:
+      - 5000:5000
+    command: flask run --host=0.0.0.0
+    volumes:
+    - .:/usr/local/app
+    environment:
+      - FLASK_APP=yaptide.application
+      - APP_SETTINGS=yaptide.settings
+      - CELERY_BROKER_URL=redis://redis:6379/0
+      - CELERY_RESULT_BACKEND=redis://redis:6379/0
+    depends_on:
+      - redis
+      - yaptide_worker
+
+  yaptide_worker:
+    build: .
```

Comment:
Current docker compose file disables ARGS in Dockerfile which are used to copy `shieldhit` binary inside.
You can use following snippet to fix it:

```
    build: 
      context: .
      args:
        SHIELDHIT_PATH: ${SHIELDHIT_PATH}
```

Then you run `docker compose` in the following way:
```
SHIELDHIT_PATH=shield_hit12a_x86_64_demo_gfortran_v0.9.2/bin/shieldhit docker-compose up -d --build
```